### DISCUSSION TITLE
Accuracy of Observations?  Is There A Better Way?

### DISCUSSION THREAD

**Discussion by BoCoKeith** [weather-gov/api#753] [General]
I've been using the NWS API to get both current observations and forecast.  (It is really great, by the way!!)  I've noticed that current observations near me are mostly from stations at airports (KBDU, for instance), but that they seem to often be missing data points or (in my opinion) providing questionable data.  FOR INSTANCE, I'm looking out the window right now and seeing "partly cloudy" skies, and the forecasts for the next few hours are also for "partly cloudy", but KDBU is reporting "Clear".  Looking at commercial sites, e.g. Wunderground, they are also reporting "partly cloudy".

Should I be looking somewhere for current observations?  Or is there some better way to get them?

**Comment by dwhitemv25** [comment]
Hm.. the last two METARs from KBDU are ..

KBDU 121935Z AUTO 34007KT 10SM CLR 28/12 A3017 RMK AO2
KBDU 121915Z AUTO 02006KT 10SM CLR 28/12 A3017 RMK AO2

So it seems the observing system there thinks skies are clear. [MesoWest](https://mesowest.utah.edu/cgi-bin/droman/meso_base_dyn.cgi?stn=kbdu) shows the station has been reporting clear skies since 9:35 MDT, just after a shower moved through.  It's likely that the equipment at KBDU can't discriminate cloud cover when clouds are slight. 

As reports are marked AUTO during the day, there is no weather staff on site at KBDU. Compare that to, say, KDEN where there is a fully-deployed sensor suite and someone there to review the sensors and adjust the reports as needed.

Complaints about observation accuracy are best addressed to your Weather Forecast Office, they would know the responsible parties to address any issues.

Be aware that other weather sites are likely using output from global weather models to give local "conditions." The accuracy of that information is wholly dependent on the accuracy of the model, and there is little to no ground truth involved.

**Comment by akrherz** [comment]
Not all ceilometers are created equal.  Sites like KBDU can only "see" up to 12,000 feet.  Whereas sites like KDEN can go higher.  So the current KDEN observation makes sense in this context

> 	KDEN 121953Z 07006KT 10SM FEW100 SCT140 SCT220
